test/fail/ok/suggest-long.tc.ok

```diff
@@ -0,0 +1,259 @@
+suggest-long.mo:3.1-3.5: type error [M0072], field stableM does not exist in type 
+  module {
```

Comment:
Would be good to have a way to shorten this error message. In practice, nobody really looks at it, because it's too long.

Reply:
The long error message might be useful for LLMs...maybe instead of showing this by default, we could gate this behind a compiler flag intended for giving additional context for AI code completion? We could also include the full [error message explanations](https://github.com/dfinity/motoko/tree/master/src/lang_utils/error_codes) and other context which could help with automatically fixing errors.

Reply:
Yes, we have `--print-source-on-error` and `--error-detail` flags already. Maybe `print-source` flags can be absorbed by `error-detail`, and `error-detail` can also control whether we list the module bindings.

Another idea is to have two error messages, one for human, one for AI...

Reply:
The error-detail flag is just for syntax errors. I've added a -ai-errors flag for now and used it to disable the suggestions.